--- ../suckless/st-0.8.5/config.h	2022-02-02 22:51:56.898028595 -0300
+++ config.h	2022-02-02 20:14:39.460647818 -0300
@@ -5,8 +5,8 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
-static int borderpx = 2;
+static char *font = "Monoid-9:antialias=true:hinting=true";
+static int borderpx = 4;

 /*
  * What program is execed by st depends of these precedence rules:
@@ -91,37 +91,33 @@
  *
  *  stty tabs
  */
-unsigned int tabspaces = 8;
+unsigned int tabspaces = 4;

 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-    /* 8 normal colors */
-    "black",
-    "red3",
-    "green3",
-    "yellow3",
-    "blue2",
-    "magenta3",
-    "cyan3",
-    "gray90",
-
-    /* 8 bright colors */
-    "gray50",
-    "red",
-    "green",
-    "yellow",
-    "#5c5cff",
-    "magenta",
-    "cyan",
-    "white",
+    /* modified gruvbox-dark-hark */
+    "#32302F",
+    "#D66E12", // D46D12
+    "#88B625",
+    "#EDB32D",
+    "#83A598",
+    "#D3869B",
+    "#8EC07C",
+    "#D8CBA5", // D5C4A1
+
+    "#756B61",
+    "#D66E12",
+    "#88B625",
+    "#EDB32D",
+    "#83A598",
+    "#D3869B",
+    "#8EC07C",
+    "#EBDBB2",

     [255] = 0,

-    /* more colors can be added after 255 to use with DefaultXX */
-    "#cccccc",
-    "#555555",
-    "gray90", /* default foreground colour */
-    "black", /* default background colour */
+    /* special colors */
+    [256] = "#1D2021", /* background */
 };


@@ -129,9 +125,9 @@
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 258;
-unsigned int defaultbg = 259;
-unsigned int defaultcs = 256;
+unsigned int defaultfg = 7;
+unsigned int defaultbg = 256;
+unsigned int defaultcs = 15;
 static unsigned int defaultrcs = 257;

 /*
